src/Weikio.ApiFramework.Plugins.AzureAD/Users/UserApi.cs

```diff
@@ -138,7 +138,13 @@ public async Task<ActionResult<UserDto>> CreateInvititation(string email, bool s
 
                 var invitationResult = await client.Invitations.Request().AddAsync(new Invitation()
                 {
-                    InvitedUserEmailAddress = email, InviteRedirectUrl = inviteRedirectUrl, SendInvitationMessage = sendInvitationMessage
+                    InvitedUserEmailAddress = email,
+                    InviteRedirectUrl = inviteRedirectUrl,
+                    SendInvitationMessage = sendInvitationMessage,
+                    InvitedUserMessageInfo = new InvitedUserMessageInfo
```

Comment:
Is it ok to always set new InvitedUserMessageInfo even if invitationLanguage was empty or null?

Reply:
Good point, I changed this so that language must not be null or empty to be set as used language.